composer/7/Dockerfile

```diff
@@ -23,6 +23,6 @@ VOLUME ["/app"]
 
 # Define working directory
 WORKDIR /app
-
+ENTRYPOINT ["composer"]
 # Set up the command arguments
 CMD ["composer"]
```

Comment:
Official composer does so, and when people want to use composer image, they expect everything working exactly like official composer so that their bash script is still working. Normally, people do something like this (from official docs)
```
composer () {
    tty=
    tty -s && tty=--tty
    docker run \
        $tty \
        --interactive \
        --rm \
        --user $(id -u):$(id -g) \
        --volume /etc/passwd:/etc/passwd:ro \
        --volume /etc/group:/etc/group:ro \
        --volume $(pwd):/app \
        composer "$@"
}
```
Now if they decide to use phalcon's composer instead, they should be able to swap out the image name and everything should still work :)